examples/http-client-loop.c

```diff
@@ -0,0 +1,68 @@
+#include <stdio.h>
+#include <assert.h>
+#include <string.h>
+
+#include "ut.h"
+
+static void response_cb(UtObject *object, UtObject *response);
+
+UtObjectRef client;
+int counter;
```

Comment:
These should be static, and initialized to NULL/0.

Reply:
yes dumb of me good spotting, I didn't really like using a global, had considered make a struct and passing it around in the callback_object. But I was mostly using client-loop for testing freeing of requests / sockets.